contracts/modules/maker/MakerV2Loan.sol

```diff
@@ -0,0 +1,626 @@
+pragma solidity ^0.5.4;
+
+import "./MakerV2Base.sol";
+import "../../defi/Loan.sol";
+import "../../defi/utils/MakerRegistry.sol";
+
+contract JugLike {
+    function drip(bytes32) external;
+}
+
+interface IUniswapFactory {
+    function getExchange(address _token) external view returns(IUniswapExchange);
+}
+
+interface IUniswapExchange {
+    function getEthToTokenOutputPrice(uint256 _tokens_bought) external view returns (uint256);
+    function getEthToTokenInputPrice(uint256 _eth_sold) external view returns (uint256);
+    function getTokenToEthOutputPrice(uint256 _eth_bought) external view returns (uint256);
+    function getTokenToEthInputPrice(uint256 _tokens_sold) external view returns (uint256);
+}
+
+/**
+ * @title MakerV2Loan
+ * @dev Module to migrate old CDPs and open and manage new CDPs.
```

Comment:
Could you had some explanation/context on why we decided to have the Maker module act as a proxy for user Vaults instead of letting them be the owner directly.

Reply:
Done